src/database/schemamanager.h

```diff
@@ -26,8 +29,15 @@ class SchemaManager {
         return m_currentVersion;
     }
 
+    /// If the current version has no backwards compatibility info, it is
+    /// assumed that it is only compatible with itself.
+    ///
+    /// Returns true if the targetVersion is greater or equal to the
+    /// minimum compatible version of the current schema
     bool isBackwardsCompatibleWithVersion(int targetVersion) const;
 
+    /// Does nothing if the versions are incompatible or the targetVersion is
```

Comment:
"Tries to update the database schema to targetVersion. Pending changes are rolled back upon failure."

Reply:
I thought you didn't want to repeat the signature in words, and isn't that the first sentence?

Reply:
Do whatever you think is right. I am out.